github3/pulls.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, dest, direction):
         if dest.get('repo'):
             self._repo_name = dest['repo'].get('name')
             self._repo_owner = dest['repo']['owner'].get('login')
+            self.repository = Repository(self.repository, self)
```

Comment:
One more suggestion.

 `self.repository = Repository(self.repository, self)`

You are referencing self.repository in the constructor. This may be a typo and I think you want to reference the dest object:

`self.repository = Repository(dest.get('repo'), self)`